client/src/components/QA/QAAddAnswerModal.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import Modal from 'react-modal';
+import axios from 'axios';
+import QuestionsAndAnswers from './QuestionsAndAnswers.jsx';
+import QACardAnswers from './QACardAnswers.jsx';
+
+// Modal.setAppElement('#app');
+if (process.env.NODE_ENV !== 'test') {
+  Modal.setAppElement('#app');
+}
+
+const QAAddAnswerModal = ({ question, productName }) => {
+
+  const [modalIsOpenAdd, setModalIsOpenAdd] = useState(false);
+  // handles onChange for inputs
+  const [answer, setAnswer] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [email, setEmail] = useState('');
+  // sets up an object with responses, will send this to POST requests
+  const [modalInfo, setModalInfo] = useState({});
+
+  return (
+    <div data-testid="qa-div-card-questions">
+      <button onClick={() => setModalIsOpenAdd(true)} className="qa-add-answer" href=""><b>[Add Answer]</b></button>
+      <Modal isOpen={modalIsOpenAdd} onRequestClose={() => setModalIsOpenAdd(false)}>
+        <h2>Submit Your Answer</h2>
+        <h3>{productName.name}: {question.question_body} </h3>
+        <form className="add-answer-form">
+          <label className="add-form-answer">Answer* </label>
+            <textarea rows="10" maxLength="1000" required onChange={() => setAnswer(event.target.value)}/><p></p>
```

Comment:
just a nit is when I see an indent, I assume that whatever is in there is a child of the above element. I would reserve indentation for child/parent relationship only otherwise it makes scanning kind of difficult